include/godot_cpp/templates/sort_array.hpp

```diff
@@ -176,14 +176,14 @@ class SortArray {
 		while (true) {
 			while (compare(p_array[p_first], p_pivot)) {
 				if (Validate) {
-					ERR_BAD_COMPARE(p_first == unmodified_last - 1);
+					ERR_BAD_COMPARE(p_first == unmodified_last - 1)
```

Comment:
This kind of semicolon is actually wanted, even if not technically needed.
This macro looks like a function call, and so, like `ERR_FAIL.*` macros in the core engine, we should force them to terminate with a semicolon.
```diff
diff --git a/include/godot_cpp/templates/sort_array.hpp b/include/godot_cpp/templates/sort_array.hpp
index 674ba3a..47d89fc 100644
--- a/include/godot_cpp/templates/sort_array.hpp
+++ b/include/godot_cpp/templates/sort_array.hpp
@@ -39,7 +39,8 @@ namespace godot {
 	if (unlikely(cond)) {                                             \
 		ERR_PRINT("bad comparison function; sorting will be broken"); \
 		break;                                                        \
-	}
+	} else                                                            \
+		((void)0)
 
 template <class T>
 struct _DefaultComparator {
```